common/src/main/java/tachyon/util/io/BufferUtils.java

```diff
@@ -69,4 +71,78 @@ public static ByteBuffer generateNewByteBufferFromThriftRPCResults(ByteBuffer da
   public static void putIntByteBuffer(ByteBuffer buf, int b) {
     buf.put((byte) (b & 0xFF));
   }
+
+  public static byte[] getIncreasingByteArray(int len) {
```

Comment:
We should document all these utilities, I've opened a ticket: https://tachyon.atlassian.net/browse/TACHYON-697